File: Examples/CSharp/Data/Processing/TracingDependents.cs
using System.IO;

using Aspose.Cells;
using System;

namespace CSharp.Data.Processing
{
    public class TracingDependents
    {
        public static void Run()
        {
            // ExStart:1
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType);
            
            Workbook workbook = new Workbook(dataDir + "Book1.xlsx");
            Cells cells = workbook.Worksheets[0].Cells;
            Cell cell = cells["B2"];

           Cell[] ret = cell.GetDependents(true);

            foreach (Cell c in cell.GetDependents(true))
            {
                Console.WriteLine(c.Name);
            }
            // ExEnd:1
            Console.ReadKey();
        }
    }
}

File: Examples/VisualBasic/Data/Processing/TracingDependents.vb
Imports System.IO

Imports Aspose.Cells
Imports System

Namespace Data.Processing
    Public Class TracingDependents
        Public Shared Sub Run()
            ' ExStart:1
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType)

            Dim workbook As New Workbook(dataDir & "Book1.xlsx")
            Dim cells As Cells = workbook.Worksheets(0).Cells
            Dim cell As Cell = cells("B2")

            Dim ret() As Cell = cell.GetDependents(True)

            For Each c As Cell In cell.GetDependents(True)
                Console.WriteLine(c.Name)
            Next c
            ' ExEnd:1
            Console.ReadKey()
        End Sub
    End Class
End Namespace
